scripts/reference.rb

```diff
@@ -11,24 +11,20 @@
   warning: '❗'
 }.freeze
 
+UTILS_MODULES = %i[ArrayUtils ModuleUtils RandomUtils Random]
+UTILS_METHODS = %i[k underscore fetch_sample rand shuffle unique luhn_check]
+
 # Get a list of sections
 def faker_modules
-  FFaker.constants.sort.map do |const|
-    mod = FFaker.const_get(const)
-    next unless mod.is_a?(Module)
-    next if mod == FFaker::ArrayUtils
-    next if mod == FFaker::ModuleUtils
-    next if mod == FFaker::RandomUtils
-    next if mod == FFaker::Random
-
-    mod
-  end.compact
+  FFaker.constants
+    .select { |const| FFaker.const_get(const.to_s).instance_of?(Module) }
+    .reject { |const| UTILS_MODULES.include?(const) }
+    .sort
```

Comment:
Unless you really want a symbol.
```suggestion
  FFaker.constants
    .map { |symbol| FFaker.const_get(const) }
    .select { |const| const.instance_of?(Module) }
    .reject { |const| UTILS_MODULES.include?(const) }
    .sort
```